src/cli/build_cli.ps1

```diff
@@ -0,0 +1,20 @@
+param (
+    [Parameter()]
+    [string] $app_dir = "$PSScriptRoot"
+)
+
+try { 
+    Push-Location
+    Set-Location "$app_dir/../pytypes"
+    python setup.py sdist bdist_wheel 
+    Copy-Item dist/*.whl "$app_dir"
+    Set-Location $app_dir
+    (Get-Content -path "requirements.txt") -replace 'onefuzztypes==0.0.0', './onefuzztypes-0.0.0-py3-none-any.whl' | Out-File -FilePath "requirements.txt" -Encoding "ascii"

```

Comment:
It would be great to use the same tools to build for developers as we use in CI.

We use `src/ci/set-versions` to set the appropriate version numbers into the packages.  Can you update this to not hardcode the versions to 0.0.0, such that we can use this in the CI?

Reply:
Added to accept version as an argument. It will append version number in cli name now.

Reply:
Without updating the package version, the exe will report the incorrect version when using `--version`

Reply:
I guess I understood it now. I am now using get-version, set-versions to set the version and then building cli. Let me know if any other change is needed.